packages/ui/src/theme/css/component/stepperField.scss

```diff
@@ -42,31 +42,29 @@
   );
 }
 
-.amplify-stepperfield__button--decrease {
+.amplify-stepperfield__button--decrease,
+.amplify-stepperfield__button--increase {
   background-color: var(
     --amplify-components-stepperfield-button-background-color
   );
-  &[data-invalid='true'] {
-    border-inline-end: none;
+  &--disabled {
+    background-color: var(--amplify-colors-background-disabled);
```

Comment:
Do we need to add a variable like `----amplify-components-stepperfield-button-background-color-disabled` ?